### DISCUSSION TITLE
Controlling visibility of a model in an iModel with multiple models

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#1754] [Q&A] [answered]
I am using classifier capabilities in iTwin.js for our reality model.  I have multiple CAD models in my imodel that are used for classification.  Depending on user choice, I use one of these models as classifier - this is same as in the classifier example in frontend-sample-showcase (and the code is taken from this example).

At start of app, all CAD models are getting loaded and are visible along with the reality model.  These have to turned off manually (using TreeView).  I would like to find out how I can turn off visibility of these CAD models at start up (and also make them visible later if requested by user) - just as the classifier example doesn't show these CAD models (was unable to find how it does this...)

Thank you

**Comment by pmconne** [comment, marked as answer]
The set of models displayed by a spatial view is controlled by its [model selector](https://www.itwinjs.org/learning/frontend/views/#modelselectors).

**Reply by ovaisa** [reply]
Thank you Paul for you guidance.  For benefit of others who might need this in the future, here is documentation on the function that enabled me to achieve this functionality.

https://www.itwinjs.org/reference/imodeljs-frontend/views/viewport/changemodeldisplay/
